packages/react/src/components/cookieConsent/languageSwitcher/LanguageSwitcherItem/LanguageSwitcherItem.module.scss

```diff
@@ -93,7 +88,7 @@
     padding: var(--spacing-2-xs);
 
     &:focus {
-      box-shadow: 0 0 0 var(--header-focus-outline-width) var(--item-focus-outline-color);
+      box-shadow: 0 0 0 3px var(--color-coat-of-arms);
```

Comment:
Do changing from variable names to color variables affect the possibility to theme the components? (If you know what I mean)

Reply:
Dunno if this is needed for shadows though?

Reply:
These are previously inherited from deprecated Navigation component and no longer would have effect here. Thus root level variables 🤔

Reply:
Ok, maybe (maybe) it then inherit from the CookieConsent but yeah it doesn't at least yer have theming in it 🤷‍♂️